.github/workflows/deploy_production.yml

```diff
@@ -0,0 +1,90 @@
+name: Deploy ASM to Production
+
+on:
+    # Run this workflow on push to master (incl PR merges)
+    push:
+        branches: [ master ]
```

Comment:
For consistency, should we make this deploy on production-release tag?

Reply:
oops, must have copied from the staging template